pkg/network/http/ebpf_gotls.go

```diff
@@ -130,7 +141,13 @@ func (p *GoTLSProgram) addInspectionResultToMap(result *bininspect.Result, binPa
 	}
 
 	// Map key is the pid, so it will be identified in the probe as the relevant data
-	pidStr := strings.Split(binPath, "/")[2]
+	splitPath := strings.Split(binPath, "/")
+	if len(splitPath) != 4 {
+		// parts should be "", "proc", "<pid>", "exe"
+		log.Error("got an unexpected path format")
```

Comment:
do we want to add the unexpect path format?

Reply:
I extended the log